plugins/modules/database/influxdb/influxdb_user.py

```diff
@@ -219,7 +224,17 @@ def main():
     grants = module.params['grants']
     influxdb = influx.InfluxDb(module)
     client = influxdb.connect_to_influxdb()
-    user = find_user(module, client, user_name)
+
+    user = None
+    try:
+        user = find_user(module, client, user_name)
+    except influx.exceptions.InfluxDBClientError as e:
+        if e.code == 403:
+            msg = json.loads(e.content)
+            if msg["error"] != INFLUX_AUTH_FIRST_USER_REQUIRED:
```

Comment:
It might make sense to add precautions in case `json.loads()` fails, or `msg["error"]` results in an exception (`msg` is not a dictionary, `'error' not in msg`).

Reply:
Done!

It should never happen, but if it does, I still add the `InfluxDBClientError` exception to the `fail_json` message as I think it can more useful to the final user to see what the server answered instead of a `KeyError` or a `JSONDecodeError`.